articles/libraries/auth0-android/index.md

```diff
@@ -77,13 +77,16 @@ The Authentication Client provides methods to authenticate the user against Auth
 AuthenticationAPIClient authentication = new AuthenticationAPIClient(account);
 ```
 
+Additionally, to make the server return Open ID Connect compliant responses we must either request an `audience` or flip the OIDC Conformant switch on in the client's dashboard Advanced OAuth settings. You can read more about that [here](https://auth0.com/docs/api-auth/intro#how-to-use-the-new-flows).
+
 ### Login with database connection
 
-Logging in with a database connection merely requires calling `login` with the user's email, password, and the name of the connection you wish to authenticate with. The response will be a Credentials object.
+Logging in with a database connection merely requires calling `login` with the user's email, password, and the name of the connection you wish to authenticate with. The response will be a Credentials object. By requesting the "userinfo" audience we guarantee an Open ID Connect compliant response from the Server.
 
 ```java
 authentication
     .login("info@auth0.com", "a secret password", "my-database-connection")
+    .setAudience("https://${account.namespace}/userinfo")
```

Comment:
This `login` method takes the OIDC flag from the `Auth0` instance (domain/clientId info). Although the audience will be set in any case, the server will only accept it for p2.